packages/fxa-auth-server/test/local/payments/paypal.js

```diff
@@ -69,12 +74,15 @@ describe('PayPalHelper', () => {
     // Make StatsD
     const statsd = { increment: sinon.spy() };
     Container.set(StatsD, statsd);
+    // Make currencyHelper
+    const currencyHelper = new CurrencyHelper(mockConfig);
+    Container.set(CurrencyHelper, currencyHelper);
     // Make PayPalHelper
-    paypalHelper = new PayPalHelper({ mockLog });
+    paypalHelper = new PayPalHelper({ log: mockLog });
```

Comment:
Doh, pretty sure I messed this up, was relying on TS to complain and no TS here. 😄